### PR TITLE
Fix small error in the README.md.

### PR DESCRIPTION
**Pull request by cecton**
Thank you for this library! :heart:

### CODE REVIEW

**Comment by cecton** [comment]
Yes on Firefox it said "invalid URL". I had to change it

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ use
 
 let program = async
 {
-   let (mut ws, _wsio) = WsMeta::connect( "127.0.0.1:3012", None ).await
+   let (mut ws, _wsio) = WsMeta::connect( "ws://127.0.0.1:3012", None ).await
```
Comment (najamelan) [inline README.md:132]: Ok, I see now. This one needs to change indeed.

File: README.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ cd ws_stream_tungstenite
 cargo run --example echo --release
 
 # in a different terminal:
-cargo run --example echo_tt --release -- "127.0.0.1:3312"
+cargo run --example echo_tt --release -- "ws://127.0.0.1:3312"
```
Comment (najamelan) [inline README.md:175]: But this one is for the server. It has to parse to `std::net::SocketAddr` so it must not have a protocol.

**Review by najamelan — CHANGES_REQUESTED** [review]
I was confused because there is changes to the server side here as well.

Can you rebase onto dev to remove all the extraneous commits and I'll merge.

File: README.md (hunk 2 above)
Comment (cecton) [reply README.md:175]: Wait... oh... sorry, I changed the wrong line!!

Comment (cecton) [inline README.md:175]:
```suggestion
cargo run --example echo_tt --release -- "127.0.0.1:3312"
```

**Comment by cecton** [comment]
> Can you rebase onto dev to remove all the extraneous commits and I'll merge.

If you do squash merge you won't have them.

**Comment by najamelan** [comment]
Looks like that worked. Thanks alot.

**Pull request merged**